Wikipedia/Code/WMFWelcomeAnimationViewControllers.swift

```diff
@@ -0,0 +1,85 @@
+class WMFWelcomeAnimationViewController: UIViewController {
+    var welcomePageType:WMFWelcomePageType = .intro
+    fileprivate var hasAlreadyAnimated = false
+    
+    open lazy var animationView: WMFWelcomeAnimationView? = nil
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        guard let animationView = animationView else {return}
+        animationView.translatesAutoresizingMaskIntoConstraints = false
+        view.wmf_addSubviewWithConstraintsToEdges(animationView)
+    }
+    
+    override func didMove(toParentViewController parent: UIViewController?) {
+        super.didMove(toParentViewController: parent)
+        view.superview?.layoutIfNeeded() // Fix for: http://stackoverflow.com/a/39614714
+        guard let animationView = animationView else {return}
```

Comment:
Same comment as above. Also update the other guards in this PR

Reply:
Fixed.